geoips/plugins/modules/readers/imerg_hdf5.py

```diff
@@ -51,7 +52,13 @@
 name = "imerg_hdf5"
 
 
-def call(fnames, metadata_only=False, chans=None, area_def=None, self_register=False):
+def call(
+    fnames,
+    metadata_only=False,
+    chans=None,
+    area_def=None,
+    self_register=False,
+):
```

Comment:
It appears that this code was formatted with black, but with different settings than other developers are using. Was that on purpose, and if so why? Else we probably should re-run black to un-un-wrap these lines so new development doesn't un-wrap them.

Reply:
Changes for this specific file are made through [this](https://github.com/NRLMMD-GEOIPS/geoips/commit/8ba693d24f180f5320f1a4e00c54adb517d382b5) commit. I have made changes to other files as well.